BayPass/APIs/GoogleMaps.swift

```diff
@@ -88,18 +90,19 @@ class GoogleMaps {
             let departureDate = Date(timeIntervalSince1970: Double(departureInterval))
             let arrivalDate = Date(timeIntervalSince1970: Double(arrivalInterval))
 
-            let lineName = lineJson["short_name"] as? String ?? ""
+            let lineName = lineJson["name"] as? String ?? ""
+            if let agencies = lineJson["agencies"] as? [String: Any] {
+                let agencyName = agencies["name"] as? String?
+                ag = Agency(rawValue: agencyName as! String) ?? Agency.zero
+            }
 
             // TODO: This section relies on getting the fare prices from firebase and the line from the API first
-            let line = Line(name: lineName, code: 232, destination: "De Anza", stops: [])
+            let line = Line(name: lineName, agency: ag, destination: "De Anza", color: #colorLiteral(red: 0.2901960784, green: 0.5647058824, blue: 0.8862745098, alpha: 1), transitMode: TransitMode.bus)
```

Comment:
Why are you using a static Line here? I imagine that we can get the Line from the name which we use as key in the Map you created with the initial API calls?